crates/cxx-qt-lib/src/gui/qfont.rs

```diff
@@ -176,6 +217,10 @@ mod ffi {
         #[rust_name = "set_fixed_pitch"]
         fn setFixedPitch(self: &mut QFont, enable: bool);
 
+        /// Sets the style strategy for the font to s.
+        #[rust_name = "set_style_strategy"]
+        fn setStyleStrategy(self: &mut QFont, s: QFontStyleStrategy);
```

Comment:
Can you please spell out the `s` -> `strategy` variable.

Just because Qt abbreviates this doesn't mean we have to :)